sphinx_source/listeler_ve_demetler.rst

```diff
@@ -2310,8 +2310,8 @@ Demetlerle Listelerin Birbirinden Farkı
 
 En başta da söylediğimiz gibi, demetlerle listeler birbirine çok benzer. Ama
 demetlerle listelerin birbirinden çok önemli bazı farkları da vardır. Bu iki
-veri tipi arasındaki en önemli fark, listelerin değiştirilebilir (*mutable*) bir
-veri tipi iken, demetlerin değiştirilemez (*immutable*) bir veri tipi olmasıdır.
+veri tipi arasındaki en önemli fark şudur; listeler değiştirilebilir (*mutable*) bir
+veri tipi iken, demetler değiştirilemez (*immutable*) bir veri tipidir.
```

Comment:
Bu değişikliklere gerek olmadığını düşünüyorum.

Reply:
Gereksiz iyelik ekleri kaldırılmış, anlatım bozukluğu giderilmiş

Reply:
Nasıl gerek yok? Anlatım bozukluğu var orda.

Reply:
Ben göremedim, biraz açıklar mısınız?

Reply:
"listeler'in' değiştirilebilir (*mutable*) bir veri tipi iken," bozukluk burdaki iyelik ekinden kaynaklanıyor. Şöyle de düzeltilebilir; 

""listelerin değiştirilebilir (*mutable*) bir veri tipi olmasıyken,"